tests/acceptance/26_cf-net/serial/cf-net_connect.cf

```diff
@@ -0,0 +1,52 @@
+body contain myshell
+{
+    !windows::
+        useshell => "useshell";
+    windows::
+        useshell => "powershell";
+}
+
+body common control
+{
+    inputs => {
+        "../../default.cf.sub",
+        "../../run_with_server.cf.sub"
+    };
+    bundlesequence => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+    methods:
+        "any" usebundle => dcs_fini("$(G.testdir)/cf-net_connect.txt");
+
+        "any" usebundle => generate_key;
+        "any" usebundle => trust_key;
+
+        "any" usebundle => start_server("$(this.promise_dirname)/cf-net_connect.cf.srv");
+}
+
+bundle agent test
+{
+    commands:
+        "$(sys.cf_net)"
+            args => " -H 127.0.0.1:15000 connect > $(G.testdir)/cf-net_connect.txt",
+            contain => myshell;
+}
+
+bundle agent check
+{
+    classes:
+        "pass" expression => regline("Connected & authenticated successfully to '127.0.0.1:15000'", "$(G.testdir)/cf-net_connect.txt");
+    methods:
+        pass::
```

Comment:
Thanks. You should comment on the last line of something since github only shows context before and not after (@jimis).